### PR TITLE
[KARAF-7096] Ensure that RMIServerImpl_Stub contains correct rmiServerHost

### PR DESCRIPTION
**Pull request by grgrzybek**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: management/server/src/main/java/org/apache/karaf/management/internal/Activator.java [hunk 1]
```diff
@@ -60,6 +60,8 @@
 
     private EventAdminLogger eventAdminLogger;
 
+    private String defaultRmiServerHostname;
```
Comment (jbonofre) [inline management/server/src/main/java/org/apache/karaf/management/internal/Activator.java:63]: I would rather use a name like `originalRmiServerHostname` more than `default`. This field is more to store existing `java.rmi.server.hostname` (system property)  more than a default value.

**Comment by grgrzybek** [comment]
works with 127.0.0.1, 0.0.0.0 and any IP

**Review by jbonofre — APPROVED** [review]
LGTM to me. I will merge when Jenkins is happy.

**Comment by grgrzybek** [comment]
thanks! If it's fine, I'll backport to 4.2.x, ok?

**Review by oscerd — APPROVED** [review]

**Comment by jbonofre** [comment]
Merged and cherry picked on `karaf-4.2.x` branch.

**Comment by grgrzybek** [comment]
Thanks for the backport!

**Pull request merged**